lib/state/revisions/actions.js

```diff
@@ -0,0 +1,3 @@
+export const setRevisionState = revisionState => ( {
+	type: 'setRevisionState', revisionState,
```

Comment:
Already checked that, but... I found it pretty much empty and just assumed this was the way to go. :D

And yeah, I'd go uppercase too.